templates/unbound.conf.erb

```diff
@@ -81,11 +81,19 @@ server:
 <% end -%>
 <% if @access.is_a? Array -%>
 <% @access.each do |acc| -%>
+<% if acc.chomp().split(' ').length != 1 -%>
+  access-control: <%= acc %>
+<% else -%>
```

Comment:
That means we don't add 'allow' if 'acc' have more than 1 words.

This allows to do thing like that :

access => ['127.0.0.1/32 allow_snoop', '124.69.127.0/22 allow', '124.69.127.0/22 deny']